pkg/cluster/node_test.go

```diff
@@ -44,25 +66,28 @@ func TestNewNode(t *testing.T) {
 }
 
 func rightNumber(nodes []*Node) bool {
-	for i := 0; i < len(nodes); i++ {
-		mnodes := nodes[i].serfServer.Members()
-		count := len(mnodes)
-		for j := 0; j < len(mnodes); j++ {
-			if mnodes[j].Status == serf.StatusLeft {
-				count--
+	for _, node := range nodes {
```

Comment:
I spent a lot of time designing this function. I'm curious to hear why you changed it.